### PR TITLE
Setting up Zingo for testing

### PR DESCRIPTION
**Pull request by juanky201271**
Some little changes to start polishing our first version of `Zingo` in Google Play:
* from zecwallet DAO to Zingo (the name of our App)
* from wallet.dat and zecwallet-lite.dat.txt to Zingo.dat and Zingo.dat.txt (the name of the file on device)
* new Copyright of 2022, with ZingoLabs.
* a couple of snapshots updated.

 PASS  __tests__/Button.snapshot.tsx
 PASS  __tests__/Components.snapshot.tsx
 PASS  __tests__/Components.unit.tsx
 PASS  __tests__/InfoModal.snapshot.tsx
 › 1 snapshot updated.
 PASS  __tests__/InfoModal.us.unit.tsx
 PASS  __tests__/About.snapshot.tsx
 › 1 snapshot updated.
 PASS  __tests__/InfoModal.es.unit.tsx
 PASS  __tests__/SettingsModal.snapshot.tsx
 PASS  __tests__/PrivKeyModal.snapshot.tsx
 PASS  __tests__/ImportKey.snapshot.tsx
 PASS  __tests__/ReceiveScreen.snapshot.tsx
 PASS  __tests__/SeedComponent.snapshot.tsx
 PASS  __tests__/RescanModal.snapshot.tsx
 PASS  __tests__/SettingsModal.unit.tsx (5.15 s)

Snapshot Summary
 › 2 snapshots updated from 2 test suites.

Test Suites: 14 passed, 14 total
Tests:       54 passed, 54 total
Snapshots:   2 updated, 17 passed, 19 total
Time:        5.894 s, estimated 16 s
Ran all test suites.

### CODE REVIEW

**Comment by zancas** [comment]
Hi @dannasessha can you sanity build and test this?

File: android/app/src/main/java/com/zecwalletmobile/RPCModule.kt [hunk 1]
```diff
@@ -34,7 +34,7 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
     @ReactMethod
     fun walletExists(promise: Promise) {
         // Check if a wallet already exists
-        val file = File(MainApplication.getAppContext()?.filesDir, "wallet.dat")
+        val file = File(MainApplication.getAppContext()?.filesDir, "Zingo.dat")
```
Comment (zancas) [inline android/app/src/main/java/com/zecwalletmobile/RPCModule.kt]: I believe that this is one of the places where the string `wallet` is appropriate.   We should match the filename that is written by `zingo-cli`.   @dannasessha can you tell us what that filename is?

File: android/app/src/main/java/com/zecwalletmobile/RPCModule.kt [hunk 2]
```diff
@@ -100,7 +100,7 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
     @ReactMethod
     fun loadExistingWallet(promise: Promise) {
         // Read the file
-        val file = MainApplication.getAppContext()!!.openFileInput("wallet.dat")
+        val file = MainApplication.getAppContext()!!.openFileInput("Zingo.dat")
```
Comment (zancas) [inline android/app/src/main/java/com/zecwalletmobile/RPCModule.kt]: Again the filename of the _wallet_ file can include the string `wallet`.   It _MUST_ match the format of `zingo-cli`.   @dannasessha is the Principle Engineer on `zingo-cli`.

File: android/app/src/main/java/com/zecwalletmobile/RPCModule.kt [hunk 3]
```diff
@@ -126,7 +126,7 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
     }
     @ReactMethod
     fun deleteExistingWallet(promise: Promise) {
-        MainApplication.getAppContext()!!.deleteFile("wallet.dat")
+        MainApplication.getAppContext()!!.deleteFile("Zingo.dat")
```
Comment (zancas) [inline android/app/src/main/java/com/zecwalletmobile/RPCModule.kt]: See previous.

File: android/app/src/main/java/com/zecwalletmobile/RPCModule.kt [hunk 4]
```diff
@@ -176,12 +176,12 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
             // Log.w("MAIN", "file size${fileBytes.size}")
 
             // Save file to disk
-            val file = MainApplication.getAppContext()?.openFileOutput("wallet.dat", Context.MODE_PRIVATE)
+            val file = MainApplication.getAppContext()?.openFileOutput("Zingo.dat", Context.MODE_PRIVATE)
```
Comment (zancas) [inline android/app/src/main/java/com/zecwalletmobile/RPCModule.kt]: See previous.

**Review by zancas — CHANGES_REQUESTED** [review]
Where this app is reading-or-writing from a _wallet_ file, that file should properly have the same name as is specified in `zingo-cli`.

File: android/app/src/main/java/com/zecwalletmobile/RPCModule.kt (hunk 1 above)
Comment (juanky201271) [reply android/app/src/main/java/com/zecwalletmobile/RPCModule.kt]: I think It is only something between the App installed (in some Android device) and the internal storage of that device. The name of this file is something you can change if you want to, all above now before the first version. It doesn't matter to me, if you prefer the old one, perfect.

File: android/app/src/main/java/com/zecwalletmobile/RPCModule.kt (hunk 2 above)
Comment (juanky201271) [reply android/app/src/main/java/com/zecwalletmobile/RPCModule.kt]: As I said, I think this name is nothing to do with zingo-cli or any other lib o piece of ZingoLabs, but @dannasessha please give us some light...

**Comment by juanky201271** [comment]
> Where this app is reading-or-writing from a _wallet_ file, that file should properly have the same name as is specified in `zingo-cli`.

mmmm, the native code of Android or/and IOS is in charge of reading-writting this file from the internal storage of the device.

**Pull request closed without merging**